index.html

```diff
@@ -3,6 +3,7 @@
 title: "Multi-node OpenStack"
 description: "MicroStack is an upstream single-node OpenStack deployment which can run directly on your workstation.
 Although made for developers, it is also suitable for edge, IoT and appliances."
+meta_copydoc: "https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit?usp=sharing"
```

Comment:
Could you update this to: https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit

Reply:
you mean get rid of usp=sharing?